contrib/android/setenv_android.sh

```diff
@@ -65,7 +84,7 @@ case "$AOSP_CPU" in
     CXXFLAGS="-march=armv7-a -mthumb -mfloat-abi=softfp -funwind-tables -fexceptions -frtti"
     ;;
 
-  armv8|armv8a|aarch64|arm64|arm64-v8a)
+  armv8*|aarch64|arm64)
```

Comment:
Was it supposed to be `arm64*` here or the change is indeed intended?
The `arm64-v8a` option is not accounted for anymore but I also see that we don't use it with travis.

Reply:
I believe Android uses `arm64` alone. But `arm64*` should be fine too. `arm64*` will help users who do something like `ANDROID_CPU=arm64-v8a`. I'd make the change if I were you.

Reply:
:+1:

Reply:
@gthess, I grabbed the change for Crypto++ at [Commit ccb3e869eddd](https://github.com/weidai11/cryptopp/commit/ccb3e869eddd). Thanks/